chart/kubeapps/templates/apprepository/deployment.yaml

```diff
@@ -89,6 +89,16 @@ spec:
             - --crontab={{ .Values.apprepository.crontab }}
             {{- end }}
             - --repos-per-namespace={{ .Values.apprepository.watchAllNamespaces }}
+              {{- if.Values.apprepository.customAnnotations }}
+                {{- range $key, $value := .Values.apprepository.customAnnotations }}
+            -  --custom-annotations={{ (print $key "=" $value) | quote }}
+                {{- end }}
+              {{- end }}
+              {{- if.Values.apprepository.customLabels }}
+                {{- range $key, $value := .Values.apprepository.customLabels }}
+            -  --custom-labels={{ (print $key "=" $value) | quote }}
+                {{- end }}
+              {{- end }}
```

Comment:
I'm passing `Values.apprepository.customAnnotations` instead of `Values.commonAnnotations`, not sure if we should support both for consistency with the expected `commonXXXX` semantics.
I went this way not to struggle with the chart templating if the PR wasn't to be accepted, but no it's has been +1, I wonder what should we do.

Reply:
> I'm passing `Values.apprepository.customAnnotations` instead of `Values.commonAnnotations`, not sure if we should support both for consistency with the expected `commonXXXX` semantics.

So `custom*` is used so that these can be specific to the apprepository controller right? Yet, you're wondering if we should also add the common ones (which the chart says will be added to every deployed resource). Yes, we probably should, but not an issue if it's not done as part of this. Fine either way.

> 
> I went this way not to struggle with the chart templating if the PR wasn't to be accepted,

What do you mean?

> but now it's has been +1, I wonder what should we do, cc @absoludity

Still +1 from me, confused if you mean you were hesitant to land given it'd changed since the review.

Reply:
Yes, maybe I didn't explain myself. I'm passing just specific labels/annotations in a dedicated field in the `values.yaml`, which is ok, as we might want these to be specific to the apprepository. However, as you mention, we probably want -also- (not instead) to have the global commons* being passed.
Since it is solely a change in the chart, let me merge the changes in this PR (since it involves coding on our side), and perform this other proposed change in another one (or even in the bitnami/charts repo).